openwisp_monitoring/measure_speed.py

```diff
@@ -0,0 +1,51 @@
+# Temporarily added to identify slow tests
```

Comment:
rename this file to `tests.py` for clarity (even if it's temporary, is good to use standard file names).